ruby/views/index.erb

```diff
@@ -272,12 +298,7 @@
         // payment token to be generated before the Link handler can be
         // initialized.
         handler = Plaid.create({
-          apiVersion: 'v2',
-          clientName: 'Plaid Quickstart',
-          env: '<%= ENV["PLAID_ENV"] %>',
-          product: products,
-          key: '<%= ENV["PLAID_PUBLIC_KEY"] %>',
-          countryCodes: '<%= ENV["PLAID_COUNTRY_CODES"] || "US,CA,GB,FR,ES" %>'.split(','),
+          ...linkHandlerCommonOptions,
```

Comment:
Same question as the node quickstart about the `get_access_token` call

Reply:
https://github.com/plaid/quickstart/pull/91#discussion_r374348837